src/object_sync/hello.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <nan.h>
+
+namespace object_sync {
+
+    /**
+     * HelloObject class
+     * This is in a header file so we can access it across other .cpp files if necessary
+     * Also, this class adheres to the rule of Zero because we define no custom destructor or copy constructor
+     */
+    class HelloObject: public Nan::ObjectWrap {
+
+        public:
+            // initializer
+            static void Init(v8::Local<v8::Object> target);
+
+            // methods required for the constructor
+            static NAN_METHOD(New);
+            //static Nan::Persistent<v8::Function> &constructor();
```

Comment:
Looks like you are getting away without it, like I mentioned above, you are not actually creating an instance of the C++ HelloWorld class and currently you are only calling static methods.

You'd need to uncomment the `Nan::Persistent<v8::Function> &constructor()` and re-introduce the wrapping at https://github.com/mapbox/node-cpp-skel/blob/7b7e67854fb2a6ea5f95fe70c4cafccf55dc450c/src/hello_world.cpp#L36-L37 if you wanted to store a property on the C++ HelloWorld class and have that accessible in the threadpool. Since this is the primary reason to have a class (to have somewhere to store data) it probably makes sense to re-introduce these things from the `master` branch.